tests/gflownet/envs/test_tests.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import inspect
+
+
+def get_current_method_name():
+    return inspect.currentframe().f_back.f_code.co_name
+
+
+# def repeat_test(times):
+#     def decorator(func):
+#         def wrapper(*args, **kwargs):
+#             for _ in range(times):
+#                 func(*args, **kwargs)
+#         return wrapper
+#     return decorator
```

Comment:
Do you want to keep these lines commented out?

Reply:
removed the whole dang file ;)